docs/file-reference.md

```diff
@@ -551,4 +550,16 @@ secrets:
 - name: wordpress
   data:
     MYSQL_ROOT_PASSWORD: YWRtaW4=
+```
+
+## Example (job)
+
+```yaml
```

Comment:
the same issues as in docs/examples/jobs/job.yaml

Reply:
fixed